tests/test_causal_model.py

```diff
@@ -144,6 +142,59 @@ def test_graph_input2(self, beta, num_instruments, num_samples, num_treatments):
         common_causes = model.get_common_causes()
         assert all(node_name in common_causes for node_name in ["X1", "X2"])
 
+    @pytest.mark.parametrize(["beta", "num_instruments", "num_samples", "num_treatments"],
+                             [(10, 1, 100, 1),])
+    def test_graph_input3(self, beta, num_instruments, num_samples, num_treatments):
```

Comment:
have you tested for dagitty text file also? good to add a text file in this folder and create a new test method.

Reply:
Yes, I did test it with text file. I will include it in a new test method.